### PR TITLE
documentation: #80 create swagger for provider login methods

### PR DESCRIPTION
**Pull request by fkula5**

### CODE REVIEW

File: app/Http/Controllers/AuthController.php [hunk 1]
```diff
@@ -27,6 +60,31 @@ public function register(RegisterRequest $request)
...
+     *    required=true,
+     *    description="Pass user credentials",
+     *    @OA\JsonContent(
+     *       required={"user","password"},
+     *       @OA\Property(property="user", type="string", format="email", example="foo_user"),
+     *       @OA\Property(property="password", type="string", format="password", example="PassWord12345"),
+     *    ),
+     * ),
+     * @OA\Response(
+     *    response=422,
```
Comment (marioooo0o) [inline app/Http/Controllers/AuthController.php]: where 200?

File: app/Http/Controllers/AuthController.php [hunk 2]
```diff
@@ -46,6 +104,30 @@ public function login(LoginRequest $request)
...
+     * @OA\RequestBody(
+     *    required=true,
+     *    description="Pass provider name",
+     *    @OA\JsonContent(
+     *       required={"provider name"},
+     *       @OA\Property(property="provider_name", type="string", example="github"),
+     *    ),
+     * ),
+     * @OA\Response(
+     *    response=422,
```
Comment (marioooo0o) [inline app/Http/Controllers/AuthController.php:130]: why 422? and where 200

File: app/Http/Controllers/AuthController.php [hunk 3]
```diff
@@ -62,6 +144,30 @@ public function redirectToProvider($provider)
...
```
Comment (marioooo0o) [inline app/Http/Controllers/AuthController.php:177]: where 200?

**Review by marioooo0o — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/AuthController.php [hunk 4]
```diff
@@ -77,6 +77,13 @@ public function register(RegisterRequest $request)
      *    ),
      * ),
      * @OA\Response(
+     *    response=200,
```
Comment (marioooo0o) [inline app/Http/Controllers/AuthController.php:73]: where 401? pleace read body of login function

**Review by marioooo0o — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/AuthController.php [hunk 5]
```diff
@@ -33,7 +33,7 @@ class AuthController extends BaseController
      *    ),
      * ),
      * @OA\Response(
-     *    response=201,
+     *    response=200,
```
Comment (marioooo0o) [inline app/Http/Controllers/AuthController.php]: Can You explain, why?

**Review by marioooo0o — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/AuthController.php (hunk 5 above)
Comment (fkula5) [reply app/Http/Controllers/AuthController.php]: Well, I forgot that we return HTTP 201 after user was successfully created

Comment (fkula5) [reply app/Http/Controllers/AuthController.php]: that is my mistake

**Review by marioooo0o — APPROVED** [review]

**Pull request merged**